_RFCs/0001-rfc-for-rfcs.md

```diff
@@ -0,0 +1,185 @@
+
+## When you need to follow this process
+[When you need to follow this process]: #when-you-need-to-follow-this-process
+
+You need to follow this process if you intend to make any change considered "substantial" to the Core-Plans repository, tooling, or maintenance processes (including the RFC process itself). What constitutes a "substantial" change may evolve over time based on the community, but initially includes the following:
+
+  - Any process or operational changes to the Core-Plans Maintenance Policies.
+  - Any package feature or rewrite that breaks backwards compatibility for current users.
+  - Any Removal of a package's features (see breaking change).
+  - Changes to the standardized operational tooling in use by Core-Plans maintainers.
+  - Any changes to the standard core-plans accepted style and baseline.
+  - Any inbound Habitat change that obligates all entries in Core-Plans to change.
+
+**NOTE:** If you have opened a single PR that changes multiple plans, it is assumed you are making a substantial change to core plans. Substantial change PRs submitted without an RFC will be closed and you will be directed to open an RFC. If this was not your intention (for example in the case of updating two interdependent packages) please open separate PRs per package change.
+
+Some changes do not require an RFC:
+
+  - Package refactors or changes that are transparent to users.
+  - Adding "Net New" software to the core repository (plans for software that doesnt currently exist in the core origin).
+  - Additions only likely to be _noticed by_ other maintainers (Invisible to target package users).
+  - Bug fixes for currently available packages.
+
+If you submit a pull request to implement a new substantial change without going through the RFC process, it may be closed with a polite request to submit an RFC first.
+
+## Before creating an RFC
+[Before creating an RFC]: #before-creating-an-rfc
+
+A hastily-proposed RFC can hurt its chances of acceptance. Low quality proposals, proposals for previously-rejected features, or those that don't fit into the near-term roadmap, may be quickly rejected, which can be demotivating for the unprepared contributor. Laying some groundwork ahead of the RFC can make the process smoother.
+
+Although there is no single way to prepare for submitting an RFC, it is generally a good idea to pursue feedback from other package developers beforehand, to ascertain that the RFC may be desirable; having a consistent impact on the project requires concerted effort toward consensus-building.
+
+The most common preparations for writing and submitting an RFC include talking the idea over in the  [#core-plans channel in slack](https://habitat-sh.slack.com/messages/C7V72D23H/) as well as filing and discussing ideas on the [Core-Plans issue tracker](https://github.com/habitat-sh/core-plans/issues)
```

Comment:
I see the issue field in the template, are we considering a GitHub issue a hard requirement for an RFC?

Reply:
Nope, that's there in case the RFC corresponds with an issue. I think for the case of core-plans it will more frequently not be the case that an issue corresponds with an inbound RFC.